CharlieBackend.AdminPanel/Views/Students/AllStudents.cshtml

```diff
@@ -109,12 +161,64 @@
 
 </div>
 
-<script type="text/javascript">
+@section Scripts{
+    <script type="text/javascript">
+
+        var isOnlyActiveStudents = false;
+
+        var switchButtons = document.querySelectorAll(".js-student-list-button");
+
+        var activeStudentsRow = document.querySelectorAll("#activeStudentsRow");
+        var inactiveStudentsRow = document.querySelectorAll("#inactiveStudentsRow");
 
-    var disableStudentId = document.querySelector("#disableStudent").getAttribute('data-item-id');
+        switchButtons.forEach(function (item) {
 
-    var confirmDisableStudent = document.querySelector("#confirmDisableStudent");
+            item.addEventListener('click', function (event) {
 
-    confirmDisableStudent.href = confirmDisableStudent.href + disableStudentId;
+                event.stopPropagation();
 
-</script>
+                if (isOnlyActiveStudents) {
+
+                    document.querySelector("#activeStudents").classList.remove('d-none')
+                    document.querySelector("#inactiveStudents").classList.add('d-none');
+
+                    activeStudentsRow.forEach(function (item) {
+                        item.classList.remove('d-none');
+                    });
+
+                    inactiveStudentsRow.forEach(function (item) {
+                        item.classList.add('d-none');
+                    });
+
+                } else {
+                    document.querySelector("#inactiveStudents").classList.remove('d-none')
+                    document.querySelector("#activeStudents").classList.add('d-none');
+
+                    inactiveStudentsRow.forEach(function (item) {
+                        item.classList.remove('d-none');
+                    });
+
+                    activeStudentsRow.forEach(function (item) {
+                        item.classList.add('d-none');
+                    });
+                }
+
+                isOnlyActiveStudents = !isOnlyActiveStudents;
+
+            }, true)
+        });
+
+
+    </script>
+
+    <script type="text/javascript">
+
+
+        var disableStudentId = document.querySelector("#disableStudent").getAttribute('data-item-id');
+
+        var confirmDisableStudent = document.querySelector("#confirmDisableStudent");
```

Comment:
i don`t see confirmDisableStudent id in your html

Reply:
is it generated by boostrap ?

Reply:
It uses a modal (lines 131-149) and the modal works with a JS script (lines 202-211) to get the student ID.